sensors/ov3660.c

```diff
@@ -18,9 +17,11 @@
 #include "freertos/task.h"
 
 #if defined(ARDUINO_ARCH_ESP32) && defined(CONFIG_ARDUHAL_ESP_LOG)
+#include "Arduino.h"
```

Comment:
I think that Arduino.h should not be needed at all. maybe I left it there while I was testing (for access to the API)